ChangeLog.md

```diff
@@ -1,6 +1,12 @@
 # Changelog for yesod-cursor
 
-## Unreleased changes
+## [Unreleased](https://github.com/freckle/yesod-page-cursor/compare/v2.0.0.9...main)
+
+- None
```

Comment:
Interesting drawback of these public-API-driven changelogs and versionings, eh?

Reply:
What do you mean?

Reply:
I just meant that, as a consumer of this library, it'd be hard to tell why we had a version bump here.

Reply:
This entry is not a version bump, it's the diff from released to `main`. The next entry (.8 .. .9) is the bump, and indicates why.

Or do I misunderstand still?

Reply:
Doh, sorry :facepalm: